modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/FDSPropsTransformer/MultiselectPicklistFDSPropsTransformer.tsx

```diff
@@ -22,5 +97,61 @@ export default function MultiselectPicklistFDSPropsTransformer({
 			}: MultiselectPicklistDataRendererProps) =>
 				value ? value.map((v) => v.name).join(', ') : '',
 		},
+		onActionDropdownItemClick({
+			action,
+			itemData,
+			loadData,
+		}: {
+			action: ActionProps;
+			itemData: ItemData;
+			loadData: voidReturn;
+		}) {
+			if (action.data.id === 'delete') {
+				openModal({
+					center: true,
+					contentComponent: ({
+						closeModal,
+					}: {
+						closeModal: voidReturn;
+					}) =>
+						ModalDeleteObjectEntry({
+							closeModal,
+							itemData,
+							loadData,
+						}),
+					id: 'deleteObjectEntry',
+					size: 'md',
+					status: 'danger',
+				});
+			}
+		},
 	};
 }
+
+type ActionProps = {
```

Comment:
@Artartur Please, define all types and interfaces at the beginning of the file and sort them by alphabetical order, while also respecting the precedence order (define type/interface before its usage).